x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/entity_store/entity_store_data_client.ts

```diff
@@ -497,6 +515,13 @@ export class EntityStoreDataClient {
         interval: enrichPolicyExecutionInterval,
       });
 
+      // this task will continuously refresh the Entity Store indices based on the Data View
+      await startEntityStoreDataViewRefreshTask({
```

Comment:
By putting this function call here, it will run once for every engine. It isn't a bug since ensuring the schedule is idempotent but unnecessary.

We also do the same thing for the `FieldRetentionEnrichTask`. So we don't have to fix it in this PR.